### PR TITLE
Fix "name" for schema-compatibility in helm-values

### PR DESCRIPTION
**Pull request by siddhant2001**

### CODE REVIEW

File: helm/values.yaml [hunk 1]
```diff
@@ -152,7 +152,7 @@ schemaCompatibility:
     repository: hypertrace/schema-compatibility
     tagOverride: ""
   config:
-    name: "schema-compatibility-configmap"
+    name: "schema-compatibility"
```
Comment (kishansairam9) [inline helm/values.yaml:155]: this shouldn't be residing here, bring it up one level and use as
```
name: {{ .Values.schemaCompatibility.name }}
```
config section should only contain what are used in configmap, anything else shouldn't be under that key, move it level above

**Review by kishansairam9 — DISMISSED** [review]
config block has some stuff which is not configs

**Review by laxmanchekka — APPROVED** [review]

File: helm/templates/schema-compatibility/cronjob.yaml [hunk 2]
```diff
@@ -28,17 +28,17 @@ spec:
...
+                name: {{ .Values.schemaCompatibility.name }}
           containers:
             - command: [ "/bin/sh", "-c" ]
               args:
                 - |
                   echo "Running python script: compatibility changer"
                   cd /opt/schema-compatibility
                   /opt/venv/bin/python -m cron-job.py
-              image: "{{ .Values.image.registry }}/{{ .Values.schemaCompatibility.image.repository }}:{{ default $.Chart.AppVersion .Values.schemaCompatibility.image.tagOverride }}"
+              image: "{{ .Values.schemaCompatibility.image.repository }}:{{ default $.Chart.AppVersion .Values.schemaCompatibility.image.tagOverride }}"
```
Comment (kishansairam9) [inline helm/templates/schema-compatibility/cronjob.yaml:39]: why this change, we need registry to handle different regions

**Review by kishansairam9 — COMMENTED** [review]
fix gha failures

File: helm/templates/schema-compatibility/cronjob.yaml (hunk 2 above)
Comment (siddhant2001) [reply helm/templates/schema-compatibility/cronjob.yaml:39]: Hypertrace repos publish to dockerhub, so `hypertrace/schema-compatibility:0.3.2` is the correct image and not `us-docker.pkg.dev/traceable-actions-386017/docker/hypertrace/schema-compatibility:0.3.2`

cc: @ravisingal

Comment (kishansairam9) [reply helm/templates/schema-compatibility/cronjob.yaml:39]: Ah okay, fine then

**Review by laxmanchekka — APPROVED** [review]
